todoapp/users/models.py

```
@@ -1,30 +1,50 @@
-    - is_staff
-    - date_joined (default should be time of object creation)
-    - last_name (max_length=150)
-    """
 
+    first_name = models.CharField("first_name", max_length=30)
+    last_name = models.CharField("last_name", max_length=150, blank=True)
+    email = models.EmailField("email address", unique=True)
+    is_staff = models.BooleanField(
+        "staff status",
+        default = False,
+        help_text="user can login to admin site or not."
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)